### PR TITLE
add imagestream+template files to repo

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@gabemontero ptal.
(this is step 1, step 2 will be updating openshift/library to pull from here, and step 3 will be updating origin to pull from openshift/library)

File: openshift/imagestreams/jenkins-centos7.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          "openshift.io/display-name": "Jenkins 2.X",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "description": "Provides a Jenkins v2.x server on CentOS 7. For more information about using this container image, including OpenShift considerations, see https://github.com/openshift/jenkins/blob/master/README.md.",
+          "iconClass": "icon-jenkins",
+          "tags": "jenkins",
+          "version": "2.x"
+        },
+        "from": {
+          "kind": "DockerImage",
+          "name": "docker.io/openshift/jenkins-2-centos7:v3.9"
```
Comment (gabemontero) [inline openshift/imagestreams/jenkins-centos7.json]: So we've got to remember to update the tag here when we cut new release branch, etc. right?

Comment (bparees) [reply openshift/imagestreams/jenkins-centos7.json]: yeah.  on the plus side it makes it all be in one place. (sort of. we'll still have to filter it down through library and into origin)

**Comment by bparees** [comment]
[merge]

**Pull request merged**